stdlib/float.mli

```diff
@@ -60,9 +60,9 @@ external div : float -> float -> float = "%divfloat"
 external fma : float -> float -> float -> float =
   "caml_fma_float" "caml_fma" [@@unboxed] [@@noalloc]
 (** A fused multiply-add [fma x y z] computes [x * y + z] with a
-   single rounding. If no hardware support is provided,
-   double-rounding issue may happen with software emulation fallback.
-   @since 4.09.0 *)
+   single rounding. Can be not fully IEEE compliant if a software
+   emulation is use in place of the hardware instruction. @since
```

Comment:
Proposal:
````
[fma x y z] returns [x * y + z], with a best effort for computing this expression with a single rounding, using either hardware instructions (providing full IEEE compliance) or a software emulation.

@since 4.08
````

Reply:
Fixed, thanks!